src/config.rs

```diff
@@ -64,19 +64,18 @@ pub struct ConfigFile {
     yay_arguments: Option<String>,
     no_retry: Option<bool>,
     run_in_tmux: Option<bool>,
-    verbose: Option<bool>,
```

Comment:
We don't have to get rid of this. We could just perform two ifs, one before the configuration is loaded, which checks `opt.verbose`, and one after that checks `config.verbose`

Reply:
hmm, but I want to call `builder.init()` before running config `load`, after running init() is there any way to update the logger?

Reply:
I think the answer is no, unless you have another idea..?

https://github.com/seanmonstar/pretty-env-logger/issues/27

Reply:
Yeah, you're correct.